test/parsing-test.js

```diff
@@ -21,49 +22,35 @@ describe('Messages parsing', function () {
     })
 
     it('Action getting', function () {
-        assert.equal(messages.getAction(msg), "+todo")
+        assert.equal(formatting.getDataMask(msg, /\+[^*\s]+/), "+todo")
```

Comment:
We are testing against one fixed message, that is exacted behavior from user. What about unexpected behaviors?? Eg:
- will have typo in `+tod` instead `+todo`
- what if not `+todo` at all, is it still logging?
- what if 2 assignee in one message Eg +todo review code @weirdGuy @zelima

Also 
- what if +todo is in the middle, at the end?
- what if no assignee 
etc... 

We need test cases for above as well